src/components/Header/AppBanner.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {Icon, Segment} from 'semantic-ui-react';
+import {useQuery} from '@apollo/react-hooks';
```

Comment:
You want to update this to import from @apollo/client - import {useQuery} from '@apollo/client';

Reply:
Oh oops, good catch thanks @XuTheBunny!